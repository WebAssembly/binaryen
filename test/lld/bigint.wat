(module
 (table $0 1 1 funcref)
 (elem (i32.const 1) $foo)
 (export "__data_end" (global $global$1))
 (global $global$0 (mut i32) (i32.const 66208))
 (global $global$1 i32 (i32.const 658))
 (func $foo (param i64) (result i64)
  (unreachable)
 )
)
