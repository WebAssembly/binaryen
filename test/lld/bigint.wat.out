(module
 (type $i64_=>_i64 (func (param i64) (result i64)))
 (type $i32_=>_i32 (func (param i32) (result i32)))
 (type $i32_i64_=>_i64 (func (param i32 i64) (result i64)))
 (table $0 1 1 funcref)
 (elem (i32.const 1) $foo)
 (global $global$0 (mut i32) (i32.const 66208))
 (global $global$1 i32 (i32.const 658))
 (export "__data_end" (global $global$1))
 (export "__growWasmMemory" (func $__growWasmMemory))
 (export "dynCall_jj" (func $dynCall_jj))
 (func $foo (param $0 i64) (result i64)
  (unreachable)
 )
 (func $__growWasmMemory (param $newSize i32) (result i32)
  (memory.grow
   (local.get $newSize)
  )
 )
 (func $dynCall_jj (param $fptr i32) (param $0 i64) (result i64)
  (call_indirect (type $i64_=>_i64)
   (local.get $0)
   (local.get $fptr)
  )
 )
)
(;
--BEGIN METADATA --
{
  "staticBump": 90,
  "tableSize": 1,
  "declares": [
  ],
  "externs": [
  ],
  "exports": [
    "__growWasmMemory",
    "dynCall_jj"
  ],
  "namedGlobals": {
    "__data_end" : "658"
  },
  "invokeFuncs": [
  ],
  "features": [
  ],
  "mainReadsParams": 0
}
-- END METADATA --
;)
